### PR TITLE
refact(RuleCard): show all OS versions at once

### LINKED ISSUE #2943: OS version is cropped when approximately more than 2 versions are provided
A few weeks ago, we created this Balrog rule that prevents Firefox users on deprecated Windows and macOS versions to be updated. The rule list 6 OS versions, but only 2 are visible: 

image

The list should have displayed:

```
Windows_NT 6.1,Windows_NT 6.2,Windows_NT 6.3,Darwin 16,Darwin 17,Darwin 18
```

Let's change the UI to show all OS versions at once.

### PR DESCRIPTION
**Pull request by CREDO23**
## This PR fixes #2943 

Now we can see all OS versions:

Screenshot from 2023-10-06 18-05-15

Screenshot from 2023-10-09 09-15-03

The RuleCard component is based on the material UI grid system, elements inside it are set in percentages, so they're always fluid and sized relative to their parent element.

### Changes made:

- Change the `osVesrion` listItem into a gridItem and give it the full width of its parent (`xs=12`) so osVesrion has enough space to be seen

### CODE REVIEW

File: ui/src/components/RuleCard/index.jsx [hunk 1]
```diff
@@ -692,6 +669,31 @@ function RuleCard({
                 )}
               </List>
             </Grid>
+            <Grid xs={12}>
+              {rule.osVersion && (
+                <ListItem className={classes.listItem}>
```
Comment (gabrielBusta) [inline ui/src/components/RuleCard/index.jsx:674]: Should the list item be in a list? Looks OK.

File: docker-compose.yml [hunk 2]
```diff
@@ -163,9 +163,9 @@ services:
       context: .
       dockerfile: Dockerfile.database
     expose:
-      - "3306"
+      - "3307"
```
Comment (gabrielBusta) [inline docker-compose.yml:166]: Could you revert the changes on this file?

**Review by gabrielBusta — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (CREDO23) [reply docker-compose.yml:166]: Sure z let me do it!
